feedback/static/html/scale_item.html

```diff
@@ -0,0 +1,13 @@
+<div class="feedback_likert_rating">
+  <label title="{scale_text}" class="feedback_likert_label">
```

Comment:
This appears to be OK.  I think it is being used in here: 
https://github.com/pmitros/FeedbackXBlock/pull/1/files#diff-4f75d70cd74efb92c6b338163f497b8fR5

So it is contained in a group.